events/api.py

```diff
@@ -8,6 +12,20 @@ class Meta:
         model = Event
         fields = '__all__'
 
+    def validate(self, data):
+        start_time = data.get('start_time', None)
+        end_time = data.get('end_time', None)
```

Comment:
`None` default value is not needed there, it is the default anyway with `get()`.